layer.yaml

```diff
@@ -50,10 +50,18 @@ defines:
     default: 'bigtop-deploy/puppet/hiera.yaml'
     description: 'Hiera Bigtop config'
 
-  bigtop_hiera_siteyaml:
+  bigtop_hieradata_path:
     type: string
-    default: 'bigtop-deploy/puppet/hieradata/site.yaml'
-    description: 'Hiera Bigtop config'
+    default: 'bigtop-deploy/puppet/hieradata/'
+    description: 'Path to hierdata .yaml files.'
+
+  bigtop_hiera_charmyaml:
```

Comment:
We use it to compose the path to the yaml files in the Bigtop class. The thing that I refactored still exists in trunk.

Is there a better way to specify the "bigtop-deploy/puppet/hieradata" path?